### PR TITLE
Drill-doors-matsoverloadstuff

### PR DESCRIPTION
**Pull request by ferrezinhre**
## Status

(pick one)

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.
## Description

This pr fixes small bug related to how drilling affect material drop form doors.

## Steps to Test or Reproduce

...

Some useful information to include:

@Vam-Jam make code.
- specific objects or behaviours affected: 28 wood form wooden doors, 40 stone from stone doors

### CODE REVIEW

**Comment by furai** [comment]
@1bardesign could you have a look a that?

File: Entities/Materials/MaterialCommon.as [hunk 1]
```diff
@@ -172,15 +172,32 @@ namespace Material
...
     // Return unless it's a harvest blob
     if (not blob.exists('harvest')) return;
-
     dictionary harvest;
     blob.get('harvest', harvest);
+    u16 maxHarvest = -1;//keep -1 unless we want a cap
+
+    if(damageBlob !is null)
+    {
+      //if we are going to have more blobs, move to switch
```
Comment (1bardesign) [inline Entities/Materials/MaterialCommon.as]: Would be great if the indentation here was kag style (tabs, equivalent to 4 wide spaces) - i know editing in github makes this annoying though.

**Comment by furai** [comment]
So should we merge that in?

**Comment by 1bardesign** [comment]
I'm fine with it (folks can talk about any better possible solution in #development), but as I said I'd prefer whoever merges it to clear up the indentation on the affected files so we at least don't go backward on code consistency.

**Comment by Vam-Jam** [comment]
closed, being moved to a different PR (requested by bunnie, and it prevents merge errors)

**Pull request closed without merging**